src/EppoClient.php

```diff
@@ -398,6 +576,7 @@ private function handleException(
         Exception $exception,
         array|bool|float|int|string|null $defaultValue
     ): array|bool|float|int|string|null {
+        // TODO: handle different exception types such as bad api key and invalid config exceptions.
```

Comment:
I'm not convinced we need to do this; I think if we're in graceful mode we want to catch all exceptions

Reply:
agreed